django_project/certification/models/certifying_organisation.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+"""
+Certifying organisation model definitions for certification apps
+"""
+
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils.text import slugify
+from core.settings.contrib import STOP_WORDS
+from unidecode import unidecode
+from course_type import CourseType
+from course import Course
+from training_center import TrainingCenter
+from course_convener import CourseConvener
+
+
+class ApprovedCertifyingOrganisationManager(models.Manager):
+    """Custom training centre manager, shows only approved training center."""
+
+    def get_queryset(self):
+        """ Query set generator. """
+
+        return super(
+            ApprovedCertifyingOrganisationManager, self).get_queryset().filter(
+                approved=True)
+
+
+class UnapprovedCertifyingOrganisationManager(models.Manager):
+    """
+    Custom training centre manager, shows only unapproved training centre.
+    """
+
+    def get_queryset(self):
+        """ Query set generator. """
+
+        return super(
+            UnapprovedCertifyingOrganisationManager, self).get_queryset(
+        ).filter(approved=False)
+
+
+class CertifyingOrganisation(models.Model):
+    """ Certifying organisation model."""
+
+    name = models.CharField(
+        help_text="Name of Organisation or Institution.",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    organisation_email = models.CharField(
+        help_text="Email address Organisation or Institution.",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    address = models.CharField(
+        help_text="Contact of Organisation or Institution.",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    organisation_phone = models.CharField(
+        help_text="Contact of Organisation or Institution.",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    approved = models.BooleanField(
+        help_text="Approval from project admin",
+        default=False
+    )
+
+    slug = models.SlugField()
+    course_type = models.ManyToManyField(CourseType)
+    course = models.ManyToManyField(Course)
+    training_center = models.ManyToManyField(TrainingCenter)
+    course_convener = models.ManyToManyField(CourseConvener)
+    objects = models.Manager()
+    approved_objects = ApprovedCertifyingOrganisationManager()
```

Comment:
Why are you adding managers to foreign objects? There must be a better way of doing this. Perhaps we could discuss what you are trying to achieve?